docs/v1/P4Runtime-Spec.adoc

```diff
@@ -2816,13 +2659,13 @@ update {
     }
   }
 }
-~ End Prototext
+----
 
-### `enum`, serializable `enum` and `error`
+==== `enum`, serializable `enum` and `error`
 
 P4~16~ supports 2 different classes of enumeration types: without underlying
 type (safe enum) and with underlying type (serializable enum or "unsafe" enum)
-[@P4Enums]. For `enum` types with no underlying type --- as well as `error` ---
+cite:[P4Enums]. For `enum` types with no underlying type --- as well as `error` ---
```

Comment:
I suspect the [Madoko?]  symbol for em-dash --- is not rendering the way we want

Reply:
According to this page on asciidoctor, it looks like an em-dash should be specified by using two dashes -- in the source file, not three.